src/foremast/utils/security_group.py

```diff
@@ -26,7 +26,7 @@
 LOG = logging.getLogger(__name__)
 
 
-@retries(max_attempts=5, wait=2, exceptions=(SpinnakerSecurityGroupError))
+@retries(max_attempts=10, wait=6, exceptions=(SpinnakerSecurityGroupError))
```

Comment:
Let's leave this alone for now, this will result in potentially 10 API operations with a delay of 6 seconds; i.e. 60 seconds worst case. In addition to hitting Spinnaker's Gate API hard, this will also make subsequent AWS calls thus manifesting the problem higher.

As I understand the issue was around ELBs not populating in the API so let's target that for now.

Reply:
I have replaced this with the backoff library per my initial recommendation in my pull request. I have changed the max attempts back to the original values per request